src/dmd/dmodule.d

```diff
@@ -750,6 +775,19 @@ extern (C++) final class Module : Package
         if (srcBuffer)
             return true; // already read
 
+        version (UseFileManager)
+        {
+            if (filemanager)
+            {
+                auto result = filemanager.opIndex(srcfile);
```

Comment:
If you're not aware, `opIndex` is a special name used to implement operator overloading [1]. It's intended to be called like this: `filemanager[srcfile]`.

[1] https://dlang.org/spec/operatoroverloading.html#array-ops